listenbrainz/webserver/static/js/src/stats/UserTopEntity.tsx

```diff
@@ -104,19 +104,12 @@ export default class UserTopEntity extends React.Component<
     return (
       <Card
         style={{
-          height: 550,
+          minHeight: 550,
           marginTop: 20,
-          display: "flex",
-          flexDirection: "column",
-          alignItems: "center",
         }}
       >
-        <h3 className="capitalize-bold">Top {entityTextOnCard}</h3>
-        <Loader
```

Comment:
I am not sure what exactly is the cause of the issue but the alignment is incorrect for the entity list.
PR -
![2020-08-14-171515_381x567_scrot](https://user-images.githubusercontent.com/25200200/90246442-471d2b80-de52-11ea-9c78-5594a5559478.png)
Production -
![2020-08-14-171505_389x567_scrot](https://user-images.githubusercontent.com/25200200/90246468-5603de00-de52-11ea-989b-a9fe7c618da1.png)

Reply:
Caught it. Flexbox helping us